protobuf/README.md

```diff
@@ -28,6 +29,7 @@ make
 
 # Re-generate PulsarApi
 cd pulsar-path/pulsar-common
-PROTOC=~/protobuf/src/protoc ./generate_protobuf.sh
+PROTOC=~/protobuf/src/protoc 
```

Comment:
If you break it in 2 lines, then you need to use `export PROTOC=...` otherwise the variable will not picked up by the next script